server/src/modules/clubs/application/use-cases/get-all-clubs/test-utils/create-clubs.ts

```diff
@@ -0,0 +1,19 @@
+import { Club } from '../../../../domain/entities/club'
+
+export const createMockClubs = (): Array<Club> => {
+  
+  const clubs: Array<Club> =[]
+  for (let _i = 1; _i <= 3; ++_i){
+    const clubResult = Club.create({
+        name: `Club Name ${_i}`,
+        description: `Club Description ${_i}`
+  })
+
+  if (clubResult.isErr()) throw new Error('Something went wrong with Club creation')
```

Comment:
i think it's ok to throw here since this is being used in a test

Reply:
I think something similar was done in the users equivalent, so I just copied it over.